doc/overview.md

```diff
@@ -0,0 +1,101 @@
+
+The role of the UK Proxy Node is to:
+ - convert eIDAS SAML to Verify SAML messages and vice versa
+ - validate all the incoming requests and outgoing responses 
+ - cryptographically sign all outgoing responses to EU services
+
+To understand how the UK Proxy Node works within the eIDAS scheme, see the GOV.UK Verify message flow document submitted for peer review. 
+
+## Components
+
+The UK Proxy Node links the GOV.UK Verify Hub with EU member state connector nodes. EU member states that can consume identity information each host a connector node. The UK hosts the GOV.UK Verify Hub. For more information, see the GOV.UK Verify Hub Technical Overview document submitted for peer review.  
+
+![](images/components.png)
+Diagram: The UK Proxy Node and communication
+
+The Proxy Node contains the following services:
+  - [Gateway](#gateway)
+  - [Eidas-Saml-Parser](#eidas-saml-parser)
+  - [Translator](#translator)
+  - [Verify-Service-Provier](#verify-service-provider)
+  - [Signing-service](#signing-service)
+
+### Gateway
+
+The Gateway service is an internet-facing component and is part of the UK Proxy Node. During a user journey, the Gateway service interacts with EU member state connector nodes and the GOV.UK Verify Hub.
+
+For requests from a connector node, the Gateway service forwards the eIDAS SAML AuthnRequest to the Eidas Saml Parser (ESP) service to perform the SAML validation. The Gateway will then use the Verify Service Provider (VSP) to generate SAML AuthnRequests which are passed back to the Gateway service before the SAML AuthnRequest is sent to the Hub.  
+
+For responses coming from the GOV.UK Verify Hub, the Gateway service forwards them to the Translator service. Responses from the Translator service are passed on to the EU member state connector node.
+
+### Eidas-Saml-Parser
+
+The Gateway Service sends the SAML AuthnRequest to the Eidas Saml Parser (ESP) service to validate the data and the signature of the request. 
+
+The ESP is the only service which can consume the countries connector metadata. The ESP will extract the Public Encryption Certificate and Destination of the Country from the metadata and send it back to the Gateway along with the RequestID and Issuer which is extracted from the SAML AuthnRequest.
+
+### Translator
+
+Part of the UK Proxy Node, the Translator service receives the GOV.UK Verify Hub responses forwarded by the Gateway service. 
+
+The Translator sends the SAML AuthnResponse to the Verify Service Provider (VSP) to perform the SAML validation. The Translator service will generate an eIDAS AuthnResponse based on the attributes it receives from the VSP. The Translator service then encrypts the signed assertion for the connector node and sends the whole eIDAS SAML response for signing by the Signing service.
```

Comment:
What's the "Signing Service"?

Reply:
Was referring to the CloudHSM client. As discussed, going to remove to prevent confusion.